maro/simulator/scenarios/cim/common.py

```diff
@@ -25,17 +26,18 @@ class Action:
     Args:
         vessel_idx (int): Which vessel will take action.
         port_idx (int): Which port will take action.
-        action_type (ActionType): Whether the action is a Load or a Discharge.
+        action_type (Optional[ActionType]): Whether the action is a Load or a Discharge.
```

Comment:
Actually the "automatically detection" is a patch for Action. I wonder whether we should make `None` action_type valid or not

Reply:
Do you mean `action_type` should always be explicitly set by callers?

Reply:
yes

Reply:
This change is a little bit large and will affect the existing callers of `Action`. I suggest we open another PR specifically for this matter, after the current one is merged.